spec/models/animal_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Animal, type: :model do
+  
+  it 'is not valid without a common name' do
+    animal_test = Animal.create latin_name: 'equus caballus', kingdom: 'mammal'
+    expect(animal_test.errors[:common_name]).to_not be_empty 
```

Comment:
Happy to see tests!!  Very cool. Since you're passing back a status code and message, it'd be good to assert on those too